src/packages/core/section/section-sidebar-context-menu/section-sidebar-context-menu.element.ts

```diff
@@ -85,7 +85,7 @@ export class UmbSectionSidebarContextMenuElement extends UmbLitElement {
 	#renderModal() {
 		return this._isOpen && this._unique !== undefined && this._entityType
 			? html`<div id="action-modal">
-					<h3>${this._headline}</h3>
+					<h3>${this.localize.string(this._headline ?? '')}</h3>
```

Comment:
Perhaps we should remove the `h3` entirely if there is no headline?

Reply:
I'll update the PR 👍